src/rules/css/detectBadMozBinding.js

```diff
@@ -2,16 +2,17 @@ import * as messages from 'messages';
 import { LOCAL_CSS_URL } from 'regex';
 
 
-export function detectBadMozBindingURL(rule) {
+export function detectBadMozBindingURL(cssRule, filename= null,
+                                       {startLine, startColumn}={}) {
   var messageList = [];
-  for (let declaration of rule.declarations) {
+  for (let declaration of cssRule.declarations) {
```

Comment:
This is probably unrelated to this PR, but it'd be nice to rename these from `cssRule` to `cssInstruction` or _something_ without rule in the name. I know it's because they're a CSS Rule, but it's conceptually hard to separate that from our own validator rules and I get confused.